aim/web/ui/src/styles/abstracts/_variables.scss

```diff
@@ -90,6 +90,7 @@ $error-color-5: #f7f1f5;
 $border-color-main: $primary-color-10;
 $border-main: 0.0625rem solid $border-color-main;
 $border-separator: 0.0625rem solid $primary-color-20;
+$border-main-darker: 0.0625rem solid $primary-color-40;
```

Comment:
can't see this variable `border-main-darker` in usage, is it necessary?
if not let's remove it

Reply:
There are a lot of variables without usage, it's our set of SCSS variables and this one is one of them that we forgot to add. I don't think that we have to remove it. Unused SCSS variables wouldn't be included in built CSS files.